samtools.1

```diff
@@ -1353,6 +1355,39 @@ paired-end reads only.
 Treat paired-end reads and single-end reads.
 .RE
 
+.TP
```

Comment:
`.TP \"-------- addreplacerg` here: aids merges (enormously!) and searching within your editor.